skops/card/_model_card.py

```diff
@@ -1300,7 +1301,11 @@ def _generate_metadata(self, metadata: ModelCardData) -> Iterator[str]:
             yield aRepr.repr(f"metadata.{key}={val},").strip('"').strip("'")
 
     def _generate_content(
-        self, data: dict[str, Section], depth: int = 1
+        self,
+        data: dict[str, Section],
+        depth: int = 1,
+        path: str | Path = "",
+        copy_files: bool = False,
```

Comment:
Personally, I'd find the implementation cleaner if we only used one argument: I would make `path` optional and if it is `None`, it means don't copy, if it is not `None`, copy to said path. There is no scenario where `path` is not `None` but `copy_files` is `True`, right?

Perhaps we can also come up with a better name than `path`, not exactly sure what, perhaps `destination_path` or something along those lines. And the docstring below should also be extended to explain the new argument (doesn't have to be a full list of parameters, just one sentence).